docs/example/tags/[tag].md

```
@@ -0,0 +1,17 @@
+---
+aside: false
+outline: false
+title: vitepress-openapi
+---
+
+<script setup lang="ts">
```

Markdownlint

Element: script
Inline HTML

(MD033, no-inline-html)